helm/mediator/templates/deployment.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Values.deployment_name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ .Values.deployment_name }}
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app: {{ .Values.deployment_name }}
+  template:
+    metadata:
+      labels:
+        app: {{ .Values.deployment_name }}
+    spec:
+      containers:
+      - name: {{ .Values.container.name }}
+        image: " {{ .Values.image.name }}:{{ .Values.image.tag }} "
+        imagePullPolicy: Always
+        ports:
+        - containerPort: {{ .Values.container.port }}
+        env:
+        - name: AGENT_PORT
+          value: "{{ .Values.environment.AGENT_PORT }}"
+        - name: AGENT_NAME
+          value: "{{ .Values.environment.AGENT_NAME }}"
+        - name: WALLET_NAME
+          value: "{{ .Values.environment.WALLET_NAME }}"
+        - name: WALLET_KEY
```

Comment:
Wallet key, postgres password, are better stored in secrets. Let's move them and reverence them accordingly:

```
          envFrom:
            - secretRef:
                name: {{ include "mediator-agent.fullname" . }}-postgresql-creds
            - secretRef:
                name: {{ include "mediator-agent.fullname" . }}-creds
```